participants/slingshot-scavenger-hunt.md

```diff
@@ -0,0 +1,96 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+**Slingshot Scavenger Hunt**
+
+Slack: @jimpick
+
+## (1) Brief Project Description
+
+Slingshot Scavenger Hunt will be a game where participants will 
+complete quests that involve finding bits of information and media
+from around the world and uploading it to the Filecoin network,
+collaboratively building hyper-local knowledge graphs.
+
+## (2) Link to Project UI
+
+* [https://sssh.v6z.me/](https://sssh.v6z.me/)
+
+## (3) What does your application/UI do?
+
+The inner gameplay loop will revolve around virtually navigating to specific
+geographic locations, and finding "wanted" information which is then
+submitted and uploaded to Filecoin. Additionally, players will
+participate in curation and retrieval tasks. As time permits, gamification
+techniques will be added to improve gameplay (leaderboards, levels, etc.)
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+As a base for our knowledge graph, we have already uploaded a 3.1GB Kiwix
+dump of the simple English Wikipedia (originally prepared for Datpedia).
+
+Potentially quests might involve finding and linking data from other curated
+datasets.
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+We may ask our users to register using GitHub accounts as we want to restrict
+access to the Lotus nodes we provide to authorized users. Apart from a small
+amount of data for things such as leaderboards, most data collected will be
+woven into a master IPLD-based knowledge graph. We will focus on public domain
+or creative commons licensed content as much as possible.
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+Probably about 10 replicas of the Kiwix dump for redundancy. Players will find and
```

Comment:
The rules will be updated to limit data to 3 replicas.  Can you change this parameter?

Reply:
The new rule is 10 replicas. I changed the parameter to 7.

Reply:
You're correct - sorry about the confusion.  Feel free to change back to 10 if preferred